### PR TITLE
Add references to issues that we are actively working on

### PR DESCRIPTION
**Pull request by samuelgoto**
***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/344.html" title="Last updated on Sep 1, 2022, 10:01 PM UTC (8c2505e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/344/9b1e8d6...8c2505e.html" title="Last updated on Sep 1, 2022, 10:01 PM UTC (8c2505e)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -1181,10 +1185,16 @@ To <dfn>fetch the accounts list</dfn> given a [=Manifest=] |manifest| and an {{I
         [=Identity Provider=]'s cookies.  This request MUST NOT follow
         [[RFC9110#header.location|HTTP redirects]] and instead return null if there is any error.
         See also [[#idp-api-accounts-endpoint]].
+
+        Issue: The credentialed fetch in this algorithm can lead to a timing attack that leaks the user's identities before the user consents. This is an active area of investigation that is being explored [here](https://github.com/fedidcg/FedCM/issues/230#issuecomment-1089040953).
+
+        Issue: This algorithm needs to use [=fetch=] appropriately, that's been tracked [here](https://github.com/fedidcg/FedCM/issues/261).
```
Comment (npm1) [inline spec/index.bs:1191]: Maybe add this where the endpoints are described (as it applies to all of them)?

File: spec/index.bs [hunk 2]
```diff
@@ -1289,14 +1299,15 @@ whether the manifest is included in the manifest list:
...
-        1. If the [=list/size=] of |json|["provider_urls"] is greater than 1, return. TODO:
-            determine the right value, probably greater than 1.
+        1. If the [=list/size=] of |json|["provider_urls"] is greater than 1, return.
         1. If |json|["provider_urls"][0] is not a [=string=], return.
         1. Set |check| to true if |json|["provider_urls"][0] [=string/is=] equal to |provider|'s
             {{IdentityProvider/configURL}}.
     1. Return |check|.
 </div>
 
+Issue: determine the size of the provider_urls array, probably [greater than 1](https://github.com/fedidcg/FedCM/issues/333).
```
Comment (npm1) [inline spec/index.bs:1309]: Can this be inside the algorithm instead?

**Review by npm1 — APPROVED** [review]

File: spec/index.bs (hunk 1 above)
Comment (samuelgoto) [reply spec/index.bs:1191]: SGTM. Done.

File: spec/index.bs (hunk 2 above)
Comment (samuelgoto) [reply spec/index.bs:1309]: Done.

**Pull request merged**